### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by sbisaria**
@tavisca-prathod

### CODE REVIEW

File: Tavisca.MobileInfoManager/MobileInfoManager.Entity/BatteryCharacterstics.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tavisca.MobileInfoManager.Entity
+{
+    public class Battery
+    {
+        public double IdleHours { get; set; }
```
Comment (tavisca-prathod) [inline Tavisca.MobileInfoManager/MobileInfoManager.Entity/BatteryCharacterstics.cs]: missing model?

File: Tavisca.MobileInfoManager/MobileInfoManager.Entity/MobileDisplayCharacterstic.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace Tavisca.MobileInfoManager.Entity
+{
+    public class MobileDisplayCharacterstic
```
Comment (tavisca-prathod) [inline Tavisca.MobileInfoManager/MobileInfoManager.Entity/MobileDisplayCharacterstic.cs]: Class name can be MobileDisplay

File: Tavisca.MobileInfoManager/MobileInfoManager.Entity/MobileDisplayCharacterstic.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    {
+        public double Size { get; set; }
+        public List<string> Colours { get; set; }
+
+        public MobileDisplayCharacterstic(double size,string colour)
+        {
+            size = Size;
+            colour = Colours;
```
Comment (tavisca-prathod) [inline Tavisca.MobileInfoManager/MobileInfoManager.Entity/MobileDisplayCharacterstic.cs]: string colour is assigned to List<string>colours.

File: Tavisca.MobileInfoManager/MobileInfoManager.Entity/Gsm.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tavisca.MobileInfoManager.Entity
+{
+    public class Gsm
+    {
+        public Battery Battery;
```
Comment (tavisca-prathod) [inline Tavisca.MobileInfoManager/MobileInfoManager.Entity/Gsm.cs]: any reason why getter and setter are not there ?

File: Tavisca.MobileInfoManager/MobileInfoManager.Entity/MobileDisplayCharacterstic.cs (hunk 3 above)
Comment (sbisaria) [inline Tavisca.MobileInfoManager/MobileInfoManager.Entity/MobileDisplayCharacterstic.cs]: should i do this:

public double Size { get; set; }
        public List<string> Colours { get; set; }

        public MobileDisplay(double size,List<string> colour)
        {
            size = Size;
            colour = Colours;
        }

Comment (sbisaria) [reply Tavisca.MobileInfoManager/MobileInfoManager.Entity/MobileDisplayCharacterstic.cs]: public double Size { get; set; }
        public List<string> Colours { get; set; }

        public MobileDisplayCharacterstic(double size, List<string> colour)
        {
            size = Size;
            colour = Colours;
        }

Comment (sbisaria) [reply Tavisca.MobileInfoManager/MobileInfoManager.Entity/MobileDisplayCharacterstic.cs]: Is this correct

Comment (tavisca-prathod) [reply Tavisca.MobileInfoManager/MobileInfoManager.Entity/MobileDisplayCharacterstic.cs]: yes

File: Tavisca.MobileInfoManager/MobileInfoManager.Entity/BatteryCharacterstics.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tavisca.MobileInfoManager.Entity
+{
+    public class BatteryCharacterstics
+    {
+        public double IdleHours { get; set; }
```
Comment (tavisca-prathod) [inline Tavisca.MobileInfoManager/MobileInfoManager.Entity/BatteryCharacterstics.cs]: You are missing Model property.

**Review by tavisca-prathod — CHANGES_REQUESTED** [review]

File: Tavisca.MobileInfoManager/MobileInfoManager.Entity/BatteryCharacterstics.cs [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tavisca.MobileInfoManager.Entity
+{
+    public class BatteryCharacterstics
+    {
+        public MobileDevice Model { get; set; }
```
Comment (tavisca-prathod) [inline Tavisca.MobileInfoManager/MobileInfoManager.Entity/BatteryCharacterstics.cs]: Why Model is of type MobileDevice ?

**Review by tavisca-prathod — CHANGES_REQUESTED** [review]

**Pull request closed without merging**